packages/components/organisms/f-cookie-banner/f-cookie-banner-static/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "description": "Fozzie Cookie Banner configuration to compile via vue cli prerender to vanilla js/css",
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "yarn install && yarn cache clean --all && yarn exec gulp"
+  },
+  "browserslist": [
+    "extends @justeat/browserslist-config-fozzie"
+  ],
+  "dependencies": {
+    "@justeat/f-cookie-banner": "file:../dist"
+  },
+  "peerDependencies": {
+    "@justeat/browserslist-config-fozzie": ">=1.1.1",
+    "js-cookie": "2.2.1"
+  },
+  "devDependencies": {
+    "@vue/cli-plugin-babel": "~4.5.0",
```

Comment:
Super small thing – we tend to lock packages down (removing `~` and `^` as it means we're less likely to hit strange bugs caused by different versions across peoples machines. Could you do that in the packages here?